### PR TITLE
Updating dependencies

### PR DESCRIPTION
**Pull request by andrewrlee**
### Context

> Does this issue have a Trello card?

https://trello.com/c/uDlwfnt7

### Intent

Updating dependencies as running a bit behind on a few major versions

> What changes are introduced by this PR that correspond to the above card?

Main things to take note of are: 
* husky required a bit of new config
* Video.js had a minor update
* The Sentry client had a major upgrade
* Removed from-html as it wasn't being used
* jquery was upgraded

> Would this PR benefit from screenshots?

### Considerations

> Is there any additional information that would help when reviewing this PR?

probably should check:
* video playback 
* double check browser compatibility for jquery update
* sentry io updates

> Are there any steps required when merging/deploying this PR?

Should monitor usage to ensure no regressions were introduced 

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/utils/enums.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = {
     PRETTY_DATE: 'EEEE d MMMM yyyy',
     PRETTY_DAY: 'EEEE',
     PRETTY_DAY_AND_MONTH: 'd MMMM',
-    PRETTY_TIME: 'h:mmaaa',
+    PRETTY_TIME: 'h:mmaa',
```
Comment (andrewrlee) [inline server/utils/enums.js:10]: this was fixed to be respected to actually produce lower case AM and PM. 
https://github.com/date-fns/date-fns/pull/2016/files

That might actually be the desired behaviour but the tests specifically stated upper case values so have maintained existing behaviour

File: package.json [hunk 2]
```diff
@@ -40,66 +41,59 @@
...
-    "eslint": "^7.15.0",
+    "cypress": "^7.1.0",
+    "cypress-cucumber-preprocessor": "^4.1.0",
+    "eslint": "^7.24.0",
     "eslint-config-airbnb-base": "^14.2.1",
-    "eslint-config-prettier": "^7.0.0",
+    "eslint-config-prettier": "^8.2.0",
     "eslint-plugin-cypress": "^2.11.2",
     "eslint-plugin-import": "^2.22.1",
-    "from-html": "^0.3.3",
```
Comment (andrewrlee) [inline package.json:91]: wasn't used so removed

File: package.json [hunk 3]
```diff
@@ -40,66 +41,59 @@
...
-    "nunjucks": "^3.2.2",
+    "nunjucks": "^3.2.3",
     "passport": "^0.4.1",
     "passport-azure-ad-oauth2": "0.0.4",
     "ramda": "^0.27.1",
-    "redis": "^3.0.2",
+    "redis": "^3.1.2",
     "uuid": "^8.3.2",
-    "video.js": "^7.10.2",
+    "video.js": "^7.11.8",
```
Comment (andrewrlee) [inline package.json:73]: probably worth checking video playback still works

File: package.json [hunk 4]
```diff
@@ -40,66 +41,59 @@
...
-      "pre-push": "npm run verify"
-    }
-  },
   "jest": {
     "verbose": true,
     "testMatch": ["**/__tests__/**/*.[jt]s?(x)"]
   },
   "dependencies": {
-    "@sentry/node": "^5.29.0",
+    "@sentry/node": "^6.3.1",
```
Comment (andrewrlee) [inline package.json:49]: "Starting from the version 6.0.0, all SDKs that support sending sessions data will do so by default."  [link](https://docs.sentry.io/product/releases/health/)

**Review by jonhindle — APPROVED** [review]
Looks fine.

**Pull request merged**